hack/make-rules/test-cmd-util.sh

```diff
@@ -4699,3 +4699,271 @@ runTests() {
     exit 1
   fi
 }
+
+run_initializer_tests() {
+  set -o nounset
+  set -o errexit
+
+  create_and_use_new_namespace
+  kube::log::status "Testing --include-uninitialized"
+
+  ### Create a deployment
+  kubectl create --request-timeout=1 -f hack/testdata/initializer-deployments.yaml 2>&1 "${kube_flags[@]}" || true
```

Comment:
From help text:
```
--request-timeout='0': The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests.
```

I don't understand why you give `--request-timeout=1`. And it's not clear to me that timeout is 1s or 1m or ...

Reply:
I think it's b/c I left a comment about it. If we don't specify a timeout, the test spend 30s here for the blocked initializer. Now I'm suspecting this 1s might cause flakes, I'm not sure making it `5s` would help prevent the flake at all.

Reply:
> I don't understand why you give --request-timeout=1

@mengqiy I've added pending initializers in the yaml, which will keep blocking until timeout. Per [discussion](https://github.com/kubernetes/kubernetes/pull/50497#discussion_r134922857).

> And it's not clear to me that timeout is 1s or 1m or ...

Just follow other commands in the [test files](https://github.com/dixudx/kubernetes/blob/d80ff0f60c7d903305c740a6271616297f1570f7/hack/make-rules/test-cmd-util.sh#L1408). Actually `--request-timeout=1` means 1sec.

Since the flag description of `--request-timeout` is quite confusing. We may consider to elaborate it.

Reply:
Make sense.